lib/redis/store/interface.rb

```diff
@@ -5,8 +5,16 @@ def get(key, options = nil)
         super(key)
       end
 
+      REDIS_SET_OPTIONS = %i(ex px nx xx keepttl).freeze
+      private_constant :REDIS_SET_OPTIONS
```

Comment:
TIL!

Reply:
Rubocop got a cop for it
https://docs.rubocop.org/rubocop/cops_style.html#styleconstantvisibility